docs/quickstart/dind.md

```diff
@@ -224,7 +225,7 @@ manager inside a system container.
 
 We will use a system container image called `nestybox/alpine-supervisord-docker:latest`.
 Nestybox DockerHub public repo. The Dockerfile, supervisord.conf, and docker-entrypoint.sh files
-can be found [here](../../sys-container/dockerfiles/alpine-supervisord-docker).
+can be found [here](https://github.com/nestybox/dockerfiles/tree/main/alpine-supervisord-docker).
```

Comment:
"/tree/main" or "/blob/main" ?

Reply:
Github by default makes use of 'tree' objects to reference directories, and it relies on 'blobs' to refer to files. This link is correct.

Reply:
Ok got it.